setup-utils/install-mxnet-osx-python.sh

```diff
@@ -362,19 +360,16 @@ download_mxnet() {
 
 	echo " "
 	echo "MXNET GIT Path = ${MXNET_GITPATH}"
-	#echo "MXNET Tag = ${MXNET_TAG}"
-	#echo "You can set \$MXNET_TAG to the appropriate github repo tag"
-	#echo "If not set, the default value used is the latest release"
-	echo " "
+	echo "MXNET Tag = ${MXNET_TAG}"
+    	echo "You can set \$MXNET_TAG to the appropriate github repo tag"
```

Comment:
Can you provide the specific command example like `export MXNET_TAG=”1.0.0.rc0”`.